### DISCUSSION TITLE
Discussion | Tuesday, 26 November 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#57] [Zoom Links]
Caawa aan isku imaano Insha'Allah!

Zoom Link:
https://us06web.zoom.us/j/81103000545?pwd=DkzLPVlZbAsEJt7C9aPVO3W1fzTZdb.1
Mentions

@dugsiiyeinc/dhaqsopay-extension-developers
@dugsiiyeinc/dhaqsopay-server-developers
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by iftiiin** [comment]
I.a eng

**Comment by shiinedev** [comment]
Hye insha`allah

**Comment by AishaKeyf** [comment]
@sharafdin  Intee saac la imaanayaa

**Comment by A-M-Ahmed** [comment]
insha allah

**Comment by Nasrah-muse** [comment]
Insha Allah Diyaar

**Comment by hayle102** [comment]
Ready

**Comment by apdyqafaar** [comment]
ready

**Comment by faaraxcabdulaahi** [comment]
> Caawa aan isku imaano Insha'Allah!
> 
> Zoom Link: https://us06web.zoom.us/j/81103000545?pwd=DkzLPVlZbAsEJt7C9aPVO3W1fzTZdb.1 Mentions
> 
> @dugsiiyeinc/dhaqsopay-extension-developers @dugsiiyeinc/dhaqsopay-server-developers @dugsiiyeinc/dhaqsopay-website-developers

Ok eng diyaar waa layahay
